includes/amp-helper-functions.php

```diff
@@ -929,9 +929,20 @@ function amp_get_content_sanitizers( $post = null ) {
 	}
 
 	if ( $amp_to_amp_linking_enabled ) {
-		$sanitizers['AMP_Link_Sanitizer'] = [
-			'paired' => ! amp_is_canonical(),
-		];
+
+		/**
+		 * Normally, when in a paired mode, links to the same origin will be to AMP.
+		 *
+		 * This allows passing URLs to exclude from having AMP-to-AMP links.
+		 *
+		 * @param string[] The URLs to exclude from having AMP-to-AMP links.
+		 */
+		$excluded_amp_links = apply_filters( 'amp_to_amp_excluded_links', [] );
```

Comment:
A subtle difference, but it is the list of URLs that is being filtered not the list of links.

```suggestion
		$excluded_amp_links = apply_filters( 'amp_to_amp_excluded_urls', [] );
```

Reply:
Sure, good point. That's changed in https://github.com/ampproject/amp-wp/pull/4146/commits/d953a27f88346787df5be725425dd45b1fee4915